sql/moz-fx-data-shared-prod/telemetry_derived/latest_versions/metadata.yaml

```diff
@@ -0,0 +1,6 @@
+friendly_name: FOG Latest Versions
```

Comment:
I don't think you need to worry about this breaking anything WRT descriptions or other metadata. You'll need to apply this ACL change manually.